locales/en-US.json

```diff
@@ -240,6 +246,14 @@
   "specialFeatures": "Special Features",
   "status": "Status",
   "studios": "Studios",
+  "username": "Username",
+  "filtersNotFound": "Unable to load filters",
+  "finish": "Finish",
+  "passwordRequired": "Password Required",
```

Comment:
Some of these aren't needed with VeeValidate.

Reply:
I've updated it to use vee-validate